endpoints/ssh_keys.yml

```
@@ -0,0 +1,21 @@
+get:
+  description: >-
+    To list all of the keys in your account, send a GET request to /v2/account/keys.
+    The response will be a JSON object with a key set to ssh_keys.  
+    The value of this will be an array of key objects, each of which contain the standard key attributes.
+  operationId: list_all_keys
+  responses:
+    '200':
+      description: >-
+        A JSON object with a key set to ssh_keys. The value is an array of key objects, each of which contain the standard key attributes.
+      headers:
+        ratelimit-limit:
+          $ref: '../components/headers.yml#/ratelimit-limit'
+        ratelimit-remaining:
+          $ref: '../components/headers.yml#/ratelimit-remaining'
+        ratelimit-reset:
+          $ref: '../components/headers.yml#/ratelimit-reset'
+      content:
+        application/json:
+          schema:
+            $ref: '../components/schemas.yml#/WrappedSSHKeys'
```

This endpoint returns a "wrapped" list of keys rather than a single wrapped key.

Something like this (but it still needs the meta object):

```suggestion
            type: "object"
            properties:
              ssh_keys:
                type: "array"
                items:
                  $ref: '../components/schemas.yml#/SSHKeys'
```

---

Good catch on the array. It's just the ssh_keys that are in the array, though, not links or meta, so the array needs to be down in the schema. I went with the `WrappedSSHKeys` name for consistency, but I think we need a different name for a different level...